pkg/manager/member/tikv_member_manager.go

```diff
@@ -433,6 +437,44 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 		return nil, fmt.Errorf("get delete slots number of statefulset %s/%s failed, err:%v", ns, setName, err)
 	}
 
+	var containers []corev1.Container
+	if tc.Spec.TiDB.ShouldSeparateRocksDBLog() {
+		// mount a shared volume and tail the RocksDB log to STDOUT using a sidecar.
+		rocksDBLogFilePath := path.Join(tikvDataVol.MountPath, "db/LOG")
```

Comment:
Is the path configurable in TiKV? What if users want to customize the path?
If yes, we have to support this as we have done for tidb slow log, you can refer to #3731.

Reply:
It is, but users rarely change it, and it is configured only by toml config file. I think it is fine to fix in the dbass.

Reply:
We also have to consider the non-dbaas users for the following cases:
1. They may want to customize the path
2. They may want to configure separate PV for the logs, especially on the public cloud where there is IO limit for one disk so users may want to use another disk to reduce the IO pressure of the data disk.

Reply:
Got, consider it is a little beyond my ability now, let me do it in the next PR.